2024/29xxx/CVE-2024-29851.json

```diff
@@ -0,0 +1,164 @@
+{
+    "dataType": "CVE_RECORD",
+    "dataVersion": "5.1",
+    "cveMetadata": {
+        "cveId": "CVE-2024-29851",
+        "assignerOrgId": "36234546-b8fa-4601-9d6f-f4e334aa8ea1",
+        "state": "PUBLISHED",
+        "assignerShortName": "hackerone",
+        "dateReserved": "2024-03-21T01:04:07.089Z",
+        "datePublished": "2024-05-22T22:55:11.273Z",
+        "dateUpdated": "2024-05-22T22:55:11.273Z"
+    },
+    "containers": {
+        "cna": {
+            "descriptions": [
+                {
+                    "lang": "en",
+                    "value": "Veeam Backup Enterprise Manager allows high-privileged users to steal NTLM hash of Enterprise manager service account."
+                }
+            ],
+            "affected": [
+                {
+                    "vendor": "Veeam",
+                    "product": "Backup & Replication",
+                    "versions": [
+                        {
+                            "version": "12.1.2.172",
+                            "status": "affected",
+                            "lessThan": "12.1.2.172",
+                            "versionType": "semver"
+                        },
+                        {
+                            "version": "11",
+                            "status": "unaffected",
+                            "lessThanOrEqual": "11",
+                            "versionType": "semver"
+                        }
+                    ]
+                }
+            ],
+            "references": [
+                {
+                    "url": "https://veeam.com/kb4581"
+                }
+            ],
+            "metrics": [
+                {
+                    "cvssV3_0": {
+                        "version": "3.0",
+                        "vectorString": "CVSS:3.0/AV:N/AC:L/PR:H/UI:N/S:U/C:H/I:H/A:H",
+                        "baseScore": 7.2,
+                        "baseSeverity": "HIGH"
+                    }
+                }
+            ],
+            "providerMetadata": {
+                "orgId": "36234546-b8fa-4601-9d6f-f4e334aa8ea1",
+                "shortName": "hackerone",
+                "dateUpdated": "2024-05-22T22:55:11.273Z"
+            }
+        },
+        "adp": [
+            {
+              "metrics": [
+                {
+                  "other": {
+                    "type": "ssvc",
+                    "content": {
+                      "id": "CVE-2024-29851",
+                      "role": "CISA Coordinator",
+                      "options": [
+                        {
+                          "Exploitation": "none"
+                        },
+                        {
+                          "Automatable": "no"
+                        },
+                        {
+                          "Technical Impact": "partial"
```

Comment:
This seems incorrect. Since the C/I/A impacts are all "High," this probably should be "Technical Impact": "total"

Reply:
See also: https://github.com/cisagov/vulnrichment/pull/62. There exists guidance that roughly, (CVSS C:H and I:H) --> (SSVC Technical Impact: Total)